src/components/VolumeSize/index.tsx

```diff
@@ -32,41 +43,42 @@ const StyledTitle = styled.span`
 `
 
 const StyledValue = styled('span', {
-  shouldForwardProp: prop => !['hasError'].includes(prop),
-})`
+  shouldForwardProp: prop => !['hasError'].includes(prop.toString()),
```

Comment:
in this file too, a simpe inequality check may be simpler & faster:
```suggestion
  shouldForwardProp: prop => prop !=='hasError',
```